Hazel/src/Platform/OpenGL/OpenGLVertexArray.cpp

```diff
@@ -50,19 +51,30 @@ namespace Hazel {
 	{
 		HZ_CORE_ASSERT(vertexBuffer->GetLayout().GetElements().size(), "Vertex Buffer has no layout!");
 
-		glBindVertexArray(m_RendererID);
-		vertexBuffer->Bind();
+		//glBindVertexArray(m_RendererID);
```

Comment:
Remove commented lines.

Reply:
Resolved in https://github.com/TheCherno/Hazel/pull/66/commits/d91579dd7ff16b35eb4a684faa738f5bf1638b3d